usecase/parser/msg.go

```diff
@@ -176,14 +177,14 @@ func ParseMsgMultiSend(
 ) ([]command.Command, []string) {
 	rawInputs, _ := parserParams.Msg["inputs"].([]interface{})
 	inputs := make([]model.MsgMultiSendInput, 0, len(rawInputs))
-	addresses := make([]string, 0, len(rawInputs))
+	possibleSignerAddresses := make([]string, 0, len(rawInputs))
 	for _, rawInput := range rawInputs {
 		input, _ := rawInput.(map[string]interface{})
 		inputs = append(inputs, model.MsgMultiSendInput{
 			Address: input["address"].(string),
 			Amount:  tmcosmosutils.MustNewCoinsFromAmountInterface(input["coins"].([]interface{})),
 		})
-		addresses = append(addresses, input["address"].(string))
```

Comment:
no need for the invalid data check here?

Reply:
oh, I add it back now.